### PR TITLE
Add permissions for e2e tests.

### PR DESCRIPTION
**Pull request by MancunianSam**
This creates two roles in the management account. Each role has
permissions to assume the corresponding role in the intg/staging
accounts and these have permission to get s3 objects in the consignment
export bucket.

### CODE REVIEW

File: modules/specific-environment-permissions/templates/jenkins_s3_export_assume_role.json.tpl [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "Version": "2012-10-17",
+
+  "Statement": [
+    {
+      "Sid": "",
+      "Effect": "Allow",
+      "Action": [
+        "sts:AssumeRole",
```
Comment (TomJKing) [inline modules/specific-environment-permissions/templates/jenkins_s3_export_assume_role.json.tpl:9]: Maybe order actions alphabetically

File: modules/specific-environment-permissions/templates/jenkins_s3_export_assume_role.json.tpl [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        "s3:GetObject"
+      ],
+      "Resource": [
+        "arn:aws:iam::${account_number}:role/TDRJenkinsS3ExportRole${stage}",
+        "arn:aws:s3:::tdr-releases-mgmt/*",
+        "arn:aws:s3:::tdr-releases-mgmt"
+      ]
+    }
+  ]
+}
```
Comment (TomJKing) [inline modules/specific-environment-permissions/templates/jenkins_s3_export_assume_role.json.tpl:21]: Missing line break

File: modules/specific-environment-permissions/templates/jenkins_s3_export_assume_role.json.tpl [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      "Sid": "",
+      "Effect": "Allow",
+      "Action": [
+        "sts:AssumeRole",
+        "s3:PutObject",
+        "s3:ListBucket",
+        "s3:GetObject"
+      ],
+      "Resource": [
+        "arn:aws:iam::${account_number}:role/TDRJenkinsS3ExportRole${stage}",
```
Comment (TomJKing) [inline modules/specific-environment-permissions/templates/jenkins_s3_export_assume_role.json.tpl:15]: Is the `stage` param being converted to title case before being passing into the template?

**Review by TomJKing — COMMENTED** [review]
Looks good. Just some very minor comments.

File: modules/specific-environment-permissions/templates/jenkins_s3_export_assume_role.json.tpl (hunk 3 above)
Comment (MancunianSam) [reply modules/specific-environment-permissions/templates/jenkins_s3_export_assume_role.json.tpl]: Yeah, it is. I've changed it to `env_title_case` to make it clearer and to match the parameter it comes from.

File: modules/specific-environment-permissions/templates/jenkins_s3_export_assume_role.json.tpl (hunk 1 above)
Comment (MancunianSam) [reply modules/specific-environment-permissions/templates/jenkins_s3_export_assume_role.json.tpl:9]: I've re-ordered them.

**Review by TomJKing — APPROVED** [review]

**Pull request merged**